packages/xo-web/src/common/editable/index.js

```diff
@@ -343,70 +337,35 @@ export class Number extends Component {
 
 export class Select extends Editable {
   static propTypes = {
-    options: PropTypes.oneOfType([PropTypes.array, PropTypes.object])
-      .isRequired,
     renderer: PropTypes.func,
-  }
-
-  componentWillReceiveProps(props) {
-    if (
-      props.value !== this.props.value ||
-      props.options !== this.props.options
-    ) {
-      this.setState({
-        valueKey: findKey(props.options, option => option === props.value),
-      })
-    }
+    value: PropTypes.oneOfType([PropTypes.oneOf([null]), PropTypes.object]),
   }
 
   get value() {
-    return this.props.options[this.state.valueKey]
-  }
-
-  _onChange = event => {
-    this.setState({ valueKey: getEventValue(event) }, this._save)
-  }
-
-  _optionToJsx = (option, key) => {
-    const { renderer } = this.props
-
-    return (
-      <option key={key} value={key}>
-        {renderer ? renderer(option) : option}
-      </option>
-    )
+    return this.state.value
   }
 
-  _onEditionMount = ref => {
-    // Seems to work in Google Chrome (not in Firefox)
-    ref && ref.dispatchEvent(new window.MouseEvent('mousedown'))
-  }
+  _onChange = value => this.setState({ value }, value && this._save)
```

Comment:
I thought you wanted to save `onBlur`?

Reply:
I think it's better to save `onChange`  so we save when we press Enter key.

Reply:
But if we want to select multiple values, it will close the edition as soon as we click one.

Reply:
I have already solved this by adding a condition in `__save`

Reply:
Oh I didn't see that. The `Editable` component is a generic (abstract-like) component that should be extendable by any editable input so it shouldn't even know about the `multi` prop.